code/modules/telesci/gps.dm

```diff
@@ -34,6 +34,8 @@ var/list/GPS_list = list()
 /obj/item/device/gps/AltClick(mob/user)
 	if(!in_range(src, user))
 		return
+	if(!user.canUseTopic(user))

```

Comment:
I'm saying it should be user.canUseTopic(src)

Nothing in any of the canUseTopic references the actual atom passed through when I last checked., so it's only for future use.